### PR TITLE
Setup sorcery pw auth

### PR DESCRIPTION
**Pull request by r3drighthand**
@giladshanan 

I switched all views (even mailers) to Slim from erb.

### CODE REVIEW

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -0,0 +1,24 @@
+doctype html
+html
+  head
+    title Social Animals
+    = csrf_meta_tags
+    = csp_meta_tag
+
+    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
```
Comment (giladshanan) [inline app/views/layouts/application.html.slim]: Single quotes

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+  
+  body
+    div id="nav"
```
Comment (giladshanan) [inline app/views/layouts/application.html.slim]: I believe slim defaults to divs, and uses css style notation for id and class. So I think you can just write ‘#nav’ here and get a div with nav id.

File: app/views/user_sessions/_form.html.slim [hunk 3]
```diff
@@ -0,0 +1,13 @@
+= form_tag user_sessions_path, :method => :post do
+  div class="field"
```
Comment (giladshanan) [inline app/views/user_sessions/_form.html.slim]: It's a matter of preference, I'm used to reading markup this way so I prefer:
```suggestion
  .field
```
Totally up to you if you want to leave this more explicit.

File: app/views/user_sessions/new.html.slim [hunk 4]
```diff
@@ -0,0 +1,5 @@
+h1 Login
+
+= render 'form'
```
Comment (giladshanan) [inline app/views/user_sessions/new.html.slim]: single quotes

**Review by giladshanan — CHANGES_REQUESTED** [review]
This looks great.  I noticed a few single quotes hanging around, but I'm not too worried about that.  When we get rubocop running that should help keep us more consistent.  

What do you think about using the slim-magic `#nav` and `.field`?

File: app/views/user_sessions/_form.html.slim (hunk 3 above)
Comment (r3drighthand) [reply app/views/user_sessions/_form.html.slim]: No, I've seen it done that way before. I guess I'm still getting comfortable with slim, but writing ".class" and "#id" makes more sense.

**Comment by r3drighthand** [comment]
I fixed everything we talked about.

**Review by giladshanan — APPROVED** [review]
Looks good 👍

**Pull request merged**